app/src/main/java/com/pizzapp/OrderSummary.java

```diff
@@ -200,6 +205,30 @@ public void launchPickupActivity(View view) {
         startActivity(intent);
     }
 
+//    public boolean onOptionsItemSelected(MenuItem item)
+//    {
+//        if (item.getItemId() == android.R.id.home)
+//        {
+//            Intent intent = NavUtils.getParentActivityIntent(this);
+//            intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP|Intent.FLAG_ACTIVITY_SINGLE_TOP);
+//            NavUtils.navigateUpTo(this, intent);
+//            return true;
+//        }
+//        return true;
+//    }
```

Comment:
are all of these lines here for a reason or can they be deleted

Reply:
You can see these lines deleted in the second commit. Can you please confirm that these lines are deleted in the second commit?